src/components/ScenarioComponents/CardsComponents/GroupFilter/FiltersContainer.tsx

```diff
@@ -26,13 +26,10 @@ interface FiltersContainerProps {
   compartmentsExpanded: boolean;
 
   /** Selected compartment */
-  selectedCompartment: string;
-
-  /** Array of compartments */
-  compartments: string[];
+  selectedCompartmentId: string | null;
 
   /** Array of filtered values */
-  filteredValues: Array<Dictionary<number> | null>;
+  filteredValues: Array<Record<string, number>>;
```

Comment:
Maybe add to the comment how this coplex type should look like?
```suggestion
  /** Array of filtered values (Items: {Compartment Name: Value}) */
  filteredValues: Array<Record<string, number>>;
```